README.md

```diff
@@ -72,16 +65,28 @@ by this point.
     cd NAB
     (sudo) pip install -r requirements.txt
 
-This will install the requirements, such as pandas and simplejson.  
+This will install the additional required modules pandas and simplejson.  
 
 
 ### Usage
 
+##### Run NAB
+
     cd /path/to/nab
-    python setup.py develop (for now)
```

Comment:
I put that in there - it's required for developing on the codebase, so mostly for us. Once we feel more comfortable about it, we could take out the `develop` portion.